modules/administration-guide/partials/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -19,13 +19,6 @@ This section describes how to use the Prometheus monitoring system to collect, s
 ====
 include::example$prometheus-config.adoc[]
 ====
-ifeval::["{project-context}" == "che"]
-+
-Latest version: link:https://raw.githubusercontent.com/eclipse-che/che-server/{pro
-
-
-d-ver-patch}/deploy/openshift/templates/monitoring/prometheus-config.yaml[example `prometheus-config.yaml` on GitHub].
-endif::[]
```

Comment:
Since `{prod-ver-patch}` is currently 7.39.1, the link:
```
https://raw.githubusercontent.com/eclipse-che/che-server/{prod-ver-patch}/deploy/openshift/templates/monitoring/prometheus-config.yaml
```
results in 404 not found: https://raw.githubusercontent.com/eclipse-che/che-server/7.39.1/deploy/openshift/templates/monitoring/prometheus-config.yaml

The latest available config is from version that I found was 7.30.2: https://raw.githubusercontent.com/eclipse-che/che-server/7.30.2/deploy/openshift/templates/monitoring/prometheus-config.yaml.